okta/idp.go

```diff
@@ -254,6 +254,7 @@ func getIdentityProviderExists(idp sdk.IdentityProvider) schema.ExistsFunc {
 func NewIdpProvisioning(d *schema.ResourceData) *sdk.IDPProvisioning {
 	return &sdk.IDPProvisioning{
 		Action: d.Get("provisioning_action").(string),
+		ProfileMaster: d.Get( "profile_master").(bool),
```

Comment:
Can you run `make fmt` and commit that? Then I think we are g2g on this. Thanks!